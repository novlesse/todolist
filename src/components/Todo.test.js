import React from "react";
import renderer from "react-test-renderer";
import Todo from "./Todo";

describe("Todo snapshots", () => {
  test("Todo snapshot test", () => {
    const component = renderer.create(<Todo />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
