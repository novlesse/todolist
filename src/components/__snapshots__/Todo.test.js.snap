// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo snapshots Todo snapshot test 1`] = `
<li
  className="todo"
>
  <div
    className="stack-small"
  >
    <div
      className="c-cb"
    >
      <input
        onChange={[Function]}
        type="checkbox"
      />
      <label
        className="todo-label"
      />
    </div>
    <div
      className="btn-group"
    >
      <button
        className="btn"
        onClick={[Function]}
        type="button"
      >
        Edit 
        <span
          className="visually-hidden"
        />
      </button>
      <button
        className="btn btn__danger"
        onClick={[Function]}
        type="button"
      >
        Delete 
        <span
          className="visually-hidden"
        />
      </button>
    </div>
  </div>
</li>
`;
